tests/sources/test_zoom.py

```diff
@@ -400,14 +401,16 @@ def get_mock(mock_response):
     return async_mock
 
 
-def setup_zoom(source, fetch_past_meeting_details=False):
-    # Set up default config with default values
-    source.configuration.set_field(name="account_id", value="123")
-    source.configuration.set_field(name="client_id", value="id@123")
-    source.configuration.set_field(name="client_secret", value="secret#123")
-    source.configuration.set_field(
-        name="fetch_past_meeting_details", value=fetch_past_meeting_details
-    )
+@asynccontextmanager
+async def create_zoom_source(fetch_past_meeting_details=False):
+    async with create_source(ZoomDataSource) as source:
+        source.configuration.get_field("account_id").value = "123"
+        source.configuration.get_field("client_id").value = "id@123"
+        source.configuration.get_field("client_secret").value = "secret#123"
+        source.configuration.get_field(
+            "fetch_past_meeting_details"
+        ).value = fetch_past_meeting_details
+        yield source
```

Comment:
I notice this is already merged so this is a bit late, but this method is still creating the source before specifying configuration. For a brief moment you have an empty configuration. It's cleaner to do it like this:

```
@asynccontextmanager
async def create_zoom_source(fetch_past_meeting_details=False):
    async with create_source(
        ZoomDataSource,
        account_id="123",
        client_id="id@123",
        client_secret="secret#123",
        fetch_past_meeting_details=fetch_past_meeting_details
    ) as source:
        yield source
```